microovn/go.mod

```diff
@@ -46,15 +46,15 @@ require (
 	github.com/sirupsen/logrus v1.9.3 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.8.4 // indirect
-	github.com/zitadel/oidc/v2 v2.7.0 // indirect
-	golang.org/x/crypto v0.12.0 // indirect
-	golang.org/x/net v0.14.0 // indirect
-	golang.org/x/oauth2 v0.11.0 // indirect
+	github.com/zitadel/oidc/v2 v2.11.0 // indirect
```

Comment:
The oid change from v2.7.0 to v2.11.0 does contain mostly dependency updates and fixes, but there are a couple of new features as well. Looks acceptable.